src/com/jetbrains/GraphicsUtils.java

```diff
@@ -72,4 +72,11 @@ public interface ConstrainableGraphics2D {
          */
         void constrain(int x, int y, int w, int h);
     }
+
+    /**
+     * Checks whether display is built-in.
+     * @param display display to check
+     * @return true when display is built-in.
```

Comment:
I think an explicit compatibility statement is needed here like "Supported only on macOS version N+" or something like that.